src/include/wt_internal.h

```diff
@@ -377,6 +377,11 @@ typedef uint64_t wt_timestamp_t;
  * DO NOT EDIT: automatically built by dist/s_typedef.
  */
 
+/* Standalone builds include fast-truncate data consistency fixes. */
+#ifdef WT_STANDALONE_BUILD
+#define WT_STANDALONE_BUILD_FT_FIX 1
```

Comment:
I'd prefer to switch this to a runtime check that is off by default for non-standalone releases. Is that feasible? I think it will make WT-9061 (testing of the backward compatibility part of this) much simpler.
cc @bvpvamsikrishna since I'm handing over that testing effort to you for now.

Reply:
I agree, this is a good idea. Fixed in a023650.